nvflare/app_common/pt/pt_fed_utils.py

```diff
@@ -165,3 +166,6 @@ def update(self, ml: ModelLearnable):
         learned_weights = ml.get(ModelLearnableKey.WEIGHTS, {})
         for k, v in learned_weights.items():
             self.var_dict[k] = v
+
+    def get_persist_model_format(self):
+        return ModelFormat.PT_CHECKPOINT
```

Comment:
Why do we need `PTModelPersistenceFormatManager`?

If only `PTFileModelPersistor` is using it, why don't we put the logic in there?

Reply:
Even though PTFileModelPersistor is only one using it now, PTModelPersistenceFormatManager is a general-purpose utility that can be used by anything.